engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -323,7 +323,7 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
     context.stack match {
       case _ :: tail =>
         typeChildren(validationContext, node, context.copy(stack = tail)) { typedParams =>
-          TypeMethodReference(reference.getName, context.stack, typedParams, typeDefinitionSet, disableMethodExecutionForUnknown) match {
+          TypeMethodReference(reference.getName, context.stack.head, typedParams, typeDefinitionSet, disableMethodExecutionForUnknown) match {
```

Comment:
let's write:
`case head :: tail => ` two lines above, and here just `head` - .head/.get are always kind of a smell :)

Reply:
Ok, already improved